### PR TITLE
prepare 1.36

### PR DESCRIPTION
**Pull request by r10s**
- [x] bump version
- [x] changelog
- [x] add a device message

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,37 @@
...
+* show non-deltachat emails by default for new installations
+  (you can change this at "Settings / Chats and Media)
+* show notifications for all accounts
+* make better use of dark/light mode in "Show full message"
+* show icon beside info messages of apps
+* resilience against outages by caching DNS results for SMTP connections
+* prefer TLS over STARTTLS during autoconfiguration, set minimum TLS version to 1.2
+* use SOCKS5 configuration also for HTTP requests
+* make invite QR codes even prettier
+* improve storage my free unused areas sooner
```
Comment (adbenitez) [inline CHANGELOG.md:19]: s/my/by/ ???

Comment (Gh0stz0x) [reply CHANGELOG.md:19]: 19 Improving storage and releasing unused areas sooner? 🤔

**Review by adbenitez — APPROVED** [review]
sorry for the wait, was translating the strings to Spanish

**Comment by r10s** [comment]
thanks for all the comments, i pushed the device message to transifex some hours ago, let's give that some more time and do a release later on

**Pull request merged**